### ISSUE TITLE
JAWS 2022 VERSION NOT ANNOUNCING LIVE REGIONS

### ISSUE THREAD

**Issue by sindhuets** [FreedomScientific/standards-support#649]
JAWS fails to announce the Live region when using the following combinations - Electron Chrome 66 + TI Calculator 2.6/2.8 + JAWS2022.

Browser: Chrome version 66.0.3359.181, Electron version 3.0.0.
TI Calculator: TI-ExamCalc CE 2.6.0.28, TI-ExamCalc CE 2.8.0.50

### Expected result

> JAWS 2022 must announce the live region.

### Actual result

> JAWS 2022 not announcing the live region updates.

**Comment by sivakusayan** [comment]
This is a duplicate of this issue: https://github.com/FreedomScientific/VFO-standards-support/issues/602

**Comment by stevefaulkner** [comment]
closing as duplicate
